{"name": "test_Get_USERINFO_by_EmailNICKNAME[myfixture0]", "status": "failed", "statusDetails": {"message": "AssertionError: assert '1' == '-1'\n  - -1\n  + 1", "trace": "myfixture = ({'authKey': '', 'mobilephone': 1888809999}, ('-1', '从customer库的Customers表读取的用户信息为空'))\n\n    @pytest.mark.get_user_info_by_mobilephone\n    @pytest.mark.flaky(reruns=1,reruns_delay=10)\n    def test_Get_USERINFO_by_EmailNICKNAME(myfixture):\n    \n        #请求\n        url=Get_USERINFO_by_Mobilephone.url\n        data=myfixture[0]\n    \n        res=request(url=url,data=data)\n>       assert res['statusCode']==myfixture[1][0]\nE       AssertionError: assert '1' == '-1'\nE         - -1\nE         + 1\n\ntestcase\\Loginapi\\Get_Userinfo_by_Mobilephone_test.py:41: AssertionError"}, "parameters": [{"name": "myfixture", "value": "({'mobilephone': 1888809999, 'authKey': ''}, ('-1', '从customer库的Customers表读取的用户信息为空'))"}], "start": 1619318490670, "stop": 1619318490732, "uuid": "7f5df03d-fc69-4ee2-98b0-bdd79ffa7e7e", "historyId": "6f00c022748a59a3ead4d68b5841ddb5", "testCaseId": "2d3041441d37da8d479a6915d32023ea", "fullName": "testcase.Loginapi.Get_Userinfo_by_Mobilephone_test#test_Get_USERINFO_by_EmailNICKNAME", "labels": [{"name": "tag", "value": "flaky(reruns=1, reruns_delay=10)"}, {"name": "tag", "value": "get_user_info_by_mobilephone"}, {"name": "parentSuite", "value": "testcase.Loginapi"}, {"name": "suite", "value": "Get_Userinfo_by_Mobilephone_test"}, {"name": "host", "value": "A06P-A90F12DAA"}, {"name": "thread", "value": "1648-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcase.Loginapi.Get_Userinfo_by_Mobilephone_test"}]}